www/codemirror/codemirror-index.html

```diff
@@ -371,6 +371,17 @@
     const applyTheme = (themeName) => {
       document.getElementById('codemirror-theme').setAttribute('href', `codemirror-${themeName}.css`)  
     }
+  
+    const toggleSyntaxColors = () => {
+        var name = document.getElementById('codemirror-theme').getAttribute('href')
+        
+        var newName = name.includes("-none") ? name.replace("-none", "") : name.substring(0, name.length - 4) + "-none.css"
+        document.getElementById('codemirror-theme').setAttribute('href', newName)
+    }
+  
+    const scaleBodyText = (textSizeAdjustment) => {
+      document.querySelector('body').style['font-size'] = (textSizeAdjustment/100.0) + 'em'
```

Comment:
One thing to maybe try would be using `-webkit-text-size-adjust` vs `font-size`. It seemed to scale things proportionally a little better for articles. Not a big deal, but may be interesting to compare the two...